package.json

```diff
@@ -20,12 +20,14 @@
     "lib"
   ],
   "main": "lib/index.js",
-  "types": "lib/index",
+  "typings": "lib/index.d.ts",
```

Comment:
Do we really need to change from `types` to `typings`?

https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html